src/test/java/org/folio/dao/DataImportQueueDaoTest.java

```diff
@@ -54,118 +66,170 @@ public void setUp() {
+    queueItemDaoImpl
+      .getQueueItemById("sample-id")
       // then
-      .onComplete(context.asyncAssertSuccess(x -> {
-        verify(postgresClient, times(1)).select(anyString(), any(Tuple.class), ArgumentMatchers.<Promise<RowSet<Row>>>any());
+      .onFailure(err -> {
+        verify(postgresClient, times(1))
+          .select(
+            anyString(),
+            any(Tuple.class),
+            ArgumentMatchers.<Promise<RowSet<Row>>>any()
+          );
         verifyNoMoreInteractions(postgresClient);
-      }));
+
+        assertThat(err, is(instanceOf(NoSuchElementException.class)));
+
+        async.complete();
+      })
+      .onSuccess(er -> context.fail("Provided ID should not exist."));
   }
 
   @Test
   public void shouldUpdateQueueItemById(TestContext context) {
     // given
     doAnswer((InvocationOnMock invocation) -> {
-      Promise<RowSet<Row>> promise = invocation.getArgument(2);
-      promise.complete(new LocalRowSet(1));
-      return null;
-    }).when(postgresClient)
-      .execute(anyString(), any(Tuple.class), ArgumentMatchers.<Promise<RowSet<Row>>>any());
+        Promise<RowSet<Row>> promise = invocation.getArgument(2);
+        promise.complete(new LocalRowSet(1));
+        return null;
+      })
+      .when(postgresClient)
+      .execute(
+        anyString(),
+        any(Tuple.class),
+        ArgumentMatchers.<Promise<RowSet<Row>>>any()
+      );
 
     // when
     DataImportQueueItem queueItem = new DataImportQueueItem();
     queueItem.setFilePath("test/file/path");
     storedItemUUID = UUID.randomUUID();
     queueItem.setId(storedItemUUID.toString());
-    queueItem.setUploadDefinitionId(UUID.randomUUID()
-      .toString());
-    queueItem.setJobExecutionId(UUID.randomUUID()
-      .toString());
+    queueItem.setUploadDefinitionId(UUID.randomUUID().toString());
+    queueItem.setJobExecutionId(UUID.randomUUID().toString());
     queueItem.setSize(1000);
     queueItem.setOriginalSize(5000);
     DateTime now = new DateTime();
     queueItem.setTimestamp(now.toString());
-    queueItemDaoImpl.updateDataImportQueueItem(queueItem)
+    queueItemDaoImpl
+      .updateDataImportQueueItem(queueItem)
       // then
-      .onComplete(context.asyncAssertSuccess(x -> {
-        verify(postgresClient, times(1)).execute(anyString(), any(Tuple.class), ArgumentMatchers.<Promise<RowSet<Row>>>any());
-        verifyNoMoreInteractions(postgresClient);
-      }));
+      .onComplete(
+        context.asyncAssertSuccess(x -> {
+          verify(postgresClient, times(1))
+            .execute(
+              startsWith("UPDATE data_import_global.queue_items SET"),
+              any(Tuple.class),
+              ArgumentMatchers.<Promise<RowSet<Row>>>any()
+            );
+          verifyNoMoreInteractions(postgresClient);
+        })
+      );
   }
+
   @Test
   public void shouldDeleteQueueItemById(TestContext context) {
     // given
-    doAnswer((InvocationOnMock invocation) -> {
```

Comment:
This is a bit backwards to the others; for this one, the `postgresClient.execute` **does** return a value, so we'll use the traditional `when(object.method(...)).thenReturn(...)`.  We only have to do it the other way around (with `do`) when the function is void